nrt/data/IEEE57/RegulationProportional/description/IEEE57.tex

```diff
@@ -0,0 +1,386 @@
+\sbBlocL{Limiter}{\tikz {\draw (-0.4,-0.4) -- (0,-0.4);\draw (0,-0.4) -- (0,0.4); \draw (0,0.4) -- (0.4,0.4); }}{Gain}
+\sbSortie[5]{S}{Limiter}
+\sbRelier[$Efd$]{Limiter}{S}
+\end{tikzpicture}
+\caption{Voltage regulator}
+\end{figure}
+
+The speed governor is as follows:
+\begin{figure}[H]
+\centering
+\begin{tikzpicture}
+\sbEntree{E}
+\sbCompSum[6]{errW}{E}{}{-}{+}{}
+\sbRelier[$omega_{Ref}$]{E}{errW}
+\sbDecaleNoeudy[4]{errW}{Omega}
+\sbRelier[$omega$]{Omega}{errW}
+\sbBloc{Gain}{$K_{Gover}$}{errW}
+\sbRelier{errW}{Gain}
+\sbCompSum{sumP}{Gain}{+}{}{+}{}
+\sbRelier{Gain}{sumP}
+\sbDecaleNoeudy[-4]{sumP}{PRef}
+\sbRelier[$P_{Ref}$]{PRef}{sumP}
+\sbBlocL{Limiter}{\tikz {\draw (-0.4,-0.4) -- (0,-0.4);\draw (0,-0.4) -- (0,0.4); \draw (0,0.4) -- (0.4,0.4); }}{sumP}
+\sbSortie[5]{S}{Limiter}
+\sbRelier[$Pm$]{Limiter}{S}
+\end{tikzpicture}
+\caption{Speed Governor}
+\end{figure}
+
+The system reference frequency omegaRef is set to 1.\\
+
+The loads are modeled as voltage-dependant loads:
+\begin{equation*}
+\begin{aligned}
+& P = P_{0} * (\frac{U}{U_{0}})^\alpha \\
+& Q = Q_{0} * (\frac{U}{U_{0}})^\alpha
+\end{aligned}
+\label{Voltage-dependant load model}
+\end{equation*}
+
+All the models parameters can be viewed into the par file available in the different test case directories.
+
+\subsection{Scenarios}
+The simulated scenarios are :
+\begin{itemize}
+\item a load variation on node 9 [\ref{LoadVariation}];
+\item a disconnection of the generator connected to node 12 [\ref{DisconnectGenerator}];
+\item a disconnection of the 6-8 line [\ref{DisconnectLine}]
+\end{itemize}
+
+\subsection{Solver}
+All scenarios are simulated with the variable time step solver IDA using the following parameters:
+\begin{itemize}
+\item $Order$=2;
+\item $Accuracy_{Rel}$=10e-4;
+\item $Accuracy_{Abs}$=10e-4;
+\end{itemize}
+
+In addition, the line disconnection is also simulated with the simplified solver using in particular the following values:
+\begin{itemize}
+\item $h_{Max}$=1s;
+\item recalculateStep=false;
+\end{itemize}
+
+The complete set of parameters used for the solvers are available into the solvers.par file in each test case directory.
+
+\newpage
+\section{Results}
+
+\subsection{Load variation}
+\label{LoadVariation}
+
+At $t=20s$, the active and reactive power set points for the load on node 9 are doubled. The load value on node 9 is initially multiplied by 2 at $t=20s$ as expected and decreases in the next instants to stabilize at a lower value (due to the voltage-dependant model used). \\
+
+To compensate for this load increase, each generator has to provide more active power, which leads to a decrease in the generator speeds due to the post-event difference between the electrical power requested by the network and the mechanical power provided by the machine. The proportional speed regulation enables to counteract this effect by increasing the mechanical power and thus helps limiting the speed deviation due to the event. \\
```

Comment:
I think we are both right because we don't exactly talk about the same active power increase (transient vs steady-state ones). Indeed, it is load increase -> instantaneous Pe increase (network equations) -> speed decrease (Pm < Pe) -> Pm increase (regulation) -> steady-state Pe increase and yes, without the regulation, you definetely don't have any steady-state electrical power increase. Anyway, that wasn't clear enough and certainly uselessly complicated: I tried to reformulate in a clearer way.